### DISCUSSION TITLE
how to use goos goarch for before hooks using goreleaser-cross:v1.17.6

### DISCUSSION THREAD

**Discussion by sandhyadalavi** [goreleaser/goreleaser#2933] [Q&A] [answered]
I'm looking for a schema if anyone can help. I want to use OS and ARCH as it seems supported by https://github.com/goreleaser/goreleaser/issues/1518 and using cross compiler goreleaser/goreleaser-cross:v1.17.6

before:
  hooks:
    env:
    goos:
      - darwin
    goarch:
      - amd64
    - make check

**Comment by caarlos0** [comment, marked as answer]
you should find in:

- https://goreleaser.com/customization/
- https://goreleaser.com/customization/build/
